test/e2e/user_workloads_test.go

```diff
@@ -71,22 +73,36 @@ func (s *userWorkloadsTestSuite) TestIdlerAndPriorityClass() {
 		err := memberAwait.WaitUntilPodsDeleted(p.Namespace, wait.WithPodName(p.Name))
 		require.NoError(s.T(), err)
 	}
+	// check notification was created and Idler condition was updated
+	_, err = memberAwait.WaitForIdler("test-idler-dev", wait.IdlerConditions(Running(), IdledNotificationCreated()))
+	require.NoError(s.T(), err)
+	_, err = hostAwait.WaitForNotificationWithName("test-idler-dev-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
+	require.NoError(s.T(), err)
 
-	// make sure that "noise" pods are still there
+	// make sure that "noise" pods are still there, and notification is not created for stage namespace
 	_, err = memberAwait.WaitForPods(idlerNoise.Name, len(podsNoise), wait.PodRunning(), wait.WithPodLabel("idler", "idler"), wait.WithSandboxPriorityClass())
 	require.NoError(s.T(), err)
 	_, err = memberAwait.WaitForPods("workloads-noise", len(externalNsPodsNoise), wait.PodRunning(), wait.WithPodLabel("idler", "idler"), wait.WithOriginalPriorityClass())
 	require.NoError(s.T(), err)
+	_, err = hostAwait.WithRetryOptions(wait.TimeoutOption(10*time.Second)).WaitForNotificationWithName("test-idler-stage-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
+	require.True(s.T(), errors.IsNotFound(err))
+
+	// Check if notification has been deleted before creating another pod
+	err = hostAwait.WaitUntilNotificationWithNameDeleted("test-idler-dev-idled")
+	require.NoError(s.T(), err)
 
 	// Create another pod and make sure it's deleted.
 	// In the tests above the Idler reconcile was triggered after we changed the Idler resource (to set a short timeout).
 	// Now we want to verify that the idler reconcile is triggered without modifying the Idler resource.
+	// Notification shouldn't be created again.
 	pod := s.createStandalonePod(idler.Name, "idler-test-pod-2")    // create just one standalone pod. No need to create all possible pod controllers which may own pods.
 	_, err = memberAwait.WaitForPod(idler.Name, "idler-test-pod-2") // pod was created
 	require.NoError(s.T(), err)
 	time.Sleep(time.Duration(2*idler.Spec.TimeoutSeconds) * time.Second)
 	err = memberAwait.WaitUntilPodDeleted(pod.Namespace, pod.Name)
 	require.NoError(s.T(), err)
+	_, err = hostAwait.WaitForNotificationWithName("test-idler-dev-idled", toolchainv1alpha1.NotificationTypeIdled, wait.UntilNotificationHasConditions(Sent()))
```

Comment:
Use a custom timeout (`wait.TimeoutOption(10*time.Second)`) to save some time on the test execution.

Reply:
done in [43d63bd](https://github.com/codeready-toolchain/toolchain-e2e/pull/560/commits/43d63bdc215c35b00065d694a62354fbce6aa696)